src/endpoints/http/collections/create_test.js

```diff
@@ -1,61 +1,28 @@
-import supertest from 'supertest';
-import express from 'express';
-import bodyParser from 'body-parser';
-import {createCollection} from './create.js';
-
-const collectionEndPoint = '/collections';
-const app = express();
-app.use(bodyParser.json());
-app.use((req, res, next) => {
-  req.db = {
-    createCollection: (name, cb) => {
-      if (name === 'fail') {
-        return cb(new Error('db error'));
-      }
-
-      cb();
-    }
-  };
-
-  req.log = {
-    debug: function() {},
-    trace: function() {}
-  };
-
-  next();
-});
-
-export function testNameParamRequired(done) {
-  createCollection(app);
-
-  supertest(app)
-    .post(collectionEndPoint)
-    .expect(400)
-    .expect(res => {
-      res.message = 'name is required';
-    })
-    .end(done);
+import assert from 'assert';
+import createCollection from './create';
+import sinon from 'sinon';
+
+const mockLogger = {
```

Comment:
I think the logger object will be reused a lot. It might be better if we create a logger that can be reused during tests. Might be just a default bunyan logger